src/nvim/buffer_defs.h

```diff
@@ -965,8 +965,8 @@ struct window_S {
                                        top of the window */
   char w_topline_was_set;           /* flag set to TRUE when topline is set,
                                        e.g. by winrestview() */
-  int w_topfill;                    /* number of filler lines above w_topline */
-  int w_old_topfill;                /* w_topfill at last redraw */
+  linenr_T w_topfill;                    /* number of filler lines above w_topline */
+  linenr_T w_old_topfill;                /* w_topfill at last redraw */
```

Comment:
Please try to keep documentation aligned, and convert to doxygen style (`///< w_topfil at last redraw`).